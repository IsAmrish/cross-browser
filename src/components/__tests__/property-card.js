import React from "react";
import renderer from "react-test-renderer";
import PropertyCard from "../property-card";

describe('ListOfPropertyCard component test', () => {
  it('Render ListOfPropertyCard', () => {
    const data = {
      "property-head": {
        "title": "abc property",
        "name": "abc"
      },
      "property-body": {
        "code": "selector {\n  -webkit-abc: hidden; /* Safari */\n  abc: hidden;\n}"
      }
    };
    const tree = renderer.create(<PropertyCard propertyHead={data["property-head"]} propertyBody={data["property-body"]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});