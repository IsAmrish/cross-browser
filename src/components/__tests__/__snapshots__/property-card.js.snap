// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListOfPropertyCard component test Render ListOfPropertyCard 1`] = `
<div
  style={
    Object {
      "border": "1px solid grey",
      "padding": "30px",
    }
  }
>
  <div
    style={
      Object {
        "background": "lightblue",
        "marginBottom": "30px",
        "padding": "16px",
      }
    }
  >
    <h2
      style={
        Object {
          "margin": "0px",
        }
      }
    >
      abc property
    </h2>
  </div>
  <div
    style={
      Object {
        "backgroundColor": "#c2ddf0",
        "padding": "16px",
      }
    }
  >
    <h3>
      How to use it
    </h3>
    <code>
      <pre
        style={
          Object {
            "backgroundColor": "#e4ecf2",
          }
        }
      >
        
selector {
  -webkit-abc: hidden; /* Safari */
  abc: hidden;
}

      </pre>
    </code>
  </div>
</div>
`;
